poh-bundle-linux-x64/oh.sh

```diff
@@ -33,12 +33,14 @@ then
     cd $POH_PATH/$MYSQL_DIR/
     rm -rf ../var/lib/mysql
     mkdir -p ../var/lib/mysql
+    mkdir -p ../var/log/mysql
```

Comment:
@mwithi duplicate command

Reply:
why duplicate? it is a different path... my.cnf expect that the path for the log file exists